pdns/saxfr.cc

```diff
@@ -129,7 +129,7 @@ try
         throw PDNSException("tcp read failed");
 
       len=ntohs(len);
-      std::unique_ptr<char[]> creply(new char[len]);
+      auto creply = std::make_unique<char[]>(len);
```

Comment:
Not your doing but switching to a `std::vector` or a `PacketBuffer` would be nice, perhaps in a follow-up PR.